english/covid-19/WIP.md

```diff
@@ -1,3 +1,4 @@
-The dungeon is building, please help us contribute it
+🚧 This area of the dungeon is currently being woked upon 🚧
+
+If you have something to add here then you can help us by [contributeing to this dungeon](https://github.com/MakeContributions/markdown-dungeon/issues/112) 🙂
```

Comment:
Please check the spelling of the hyperlink: `contributeing` should be `contributing`. 😉

Reply:
will do